src/tarball.py

```diff
@@ -58,6 +58,21 @@ def _find_build_config(self, node):
             if config.tree.name == tree and config.branch == branch:
                 return config
 
+    def _find_build_commit(self, node):
+        revision = node['data'].get('kernel_revision')
```

Comment:
Is it 100% guaranteed that the node data has `kernel_revision`?

Reply:
Yes, otherwise checkout node doesn't make sense

Reply:
Alright then (I'm not familiar with those early pipeline stages)